packages/app/src/App.module.css

```diff
@@ -51,6 +51,9 @@
   --h5w-domainSlider-histogram--color: var(--secondary-dark);
   --h5w-domainSlider-histogram-indicator--color: var(--secondary-darker);
   --h5w-interactionHelp-shortcut--bgColor: var(--primary-light);
+
+  --h5w-line--color: darkblue;
+  --h5w-line--colorAux: orangered, forestgreen, red, mediumorchid, olive;
```

Comment:
I am reverting part of https://github.com/silx-kit/h5web/pull/1008 by setting back the line color in the app.

In fact, I would argue that the `useCustomColors` hook with dark fallbacks implemented in #1008 should be removed. The dark mode switch is handled at the `app` level while the hook works at the `lib` level.

I have the opinion that the vis (that is `lib`) should be completely oblivious to dark mode as it should be the responsibility of `App` (or of the `App` consumer) to implement it if needed. Further implementations deeper in the hierarchy are not difficult to discover/debug and get in the way of the global handling.

Reply:
Given, we still need to use a hook to get the CSS values in JS, I think we can leave the fallback in JS via `useCustomColors`.

I will remove the `darkFallbackk` though as discussed.